.eslintrc.cjs

```diff
@@ -159,8 +159,11 @@ module.exports = {
         allows: ["dotAll"],
       },
     ],
+    /* cspell:disable-next-line */
```

Comment:
I've seen us adding infrequent works to `cpell.json`, e.g. `lcov` (used only once). I see [only one](https://github.com/prettier/prettier/blob/9c45de68425a4328980a849c2e94b6ea66772c00/tests/integration/__tests__/early-exit.js#L56) `cspell:disable-next-line` in the codebase apart from this one, what's the reason for using it here?

Reply:
It's like `eslint-disable`, there are multiple ways to ignore the error, I don't think it matters.